README.md

```diff
@@ -10,6 +11,16 @@ This library is based on [AWS Interceptor](https://github.com/awslabs/aws-reques
 
 ## Usage
 
+Add [io.github.acm19.aws-request-signing-apache-interceptor](https://repo1.maven.org/maven2/io/github/acm19/aws-request-signing-apache-interceptor/) as a dependency.
+
+```xml
+<dependency>
+  <groupId>io.github.acm19</groupId>
+  <artifactId>aws-request-signing-apache-interceptor</artifactId>
+  <version>2.1.1</version>
```

Comment:
It's important that we update this as part of the release process, in `update_release_files_and_commit` recipe for example. A simple `sed` would do the trick.

Reply:
I knew you were going to say that. Updated. Also fixed CHANGELOG getting a -SNAPSHOT version and when you try to release it leaves -SNAPSHOT in there.

Reply:
:) That was kind of intentional.

Reply:
I think it's great now btw, I'd only leave the snapshot on the next working version comment, but I don't feel strong about it, so feel free to merge it.

Reply:
Yeah, I kinda prefer not to have SNAPSHOT in it because the actual version we plan to develop is not -SNAPSHOT.